### ISSUE TITLE
[weekly reports] Grouping results into days does not correspond to daily workflow runs

### ISSUE THREAD

**Issue by rvykydal** [rhinstaller/kickstart-tests#701]
In the Weekly summary https://github.com/rhinstaller/kickstart-tests/actions/runs/2088552154
we can se that for example `03/30/2022` the results of the scenarios come from these two runs:
- daily-iso: Daily run #504 (https://github.com/rhinstaller/kickstart-tests/actions/runs/2067880057)
- rhel8: Daily run #503 (https://github.com/rhinstaller/kickstart-tests/actions/runs/2061592448) 
- rhel9: Daily run #504 (https://github.com/rhinstaller/kickstart-tests/actions/runs/2067880057)

As the artifact to be tested is downloaded/created at the beginning of the scenario grouping by the scenario(s) run seems to be neede here. For example the `03/30/2022` should contain results for workflow run (3 scenarios) which was run on 03/30/2022.

So maybe we need to take into account the scheduled time of the day of the run (https://github.com/rhinstaller/kickstart-tests/blob/e059ddb561b36c58f4272366ea4cf525b8e9ddf0/.github/workflows/scenarios.yml#L6) (defined statically or passed dynamically) and group the results within 24 hrs from the start. Or pass the real workflow start time by the --scenario identifier and use that?

**Comment by bcl** [comment]
I'm not sure if I understand what you mean. Are you saying that it used the wrong rhel8 result?
The way it groups the artifacts is by start time returned by the GitHub artifact API. I tried to figure out a way to associate them with an action run number, but there doesn't seem to be any API calls for that.

I'm now also realizing that it is really hard to figure out which run the summary is referring to.

**Comment by rvykydal** [comment]
Yes, to put it simply we want the daily results to be grouped by the workflow run. Which turns out to be not so obvious, maybe we will need to pass some info for grouping explicitly ? (To artifact name, metadata if possible, I'd like to avoid passing it through the launcher...) I would certainly perfer to use some way via GitHub API if possible, but as you say it may be not the case.

**Comment by rvykydal** [comment]
The issue seem to be no more present.
